composeApp/src/androidMain/kotlin/it/fast4x/rimusic/service/modern/PlayerServiceModern.kt

```diff
@@ -437,6 +438,7 @@ class PlayerServiceModern : MediaLibraryService(),
         // Load persistent queue when start activity and save periodically in background
         if (isPersistentQueueEnabled) {
             maybeRestorePlayerQueue()
```

Comment:
I've removed this line because not necessary, it's just called by line 443

Reply:
Where is it called? line 443 is `val scheduler = Executors.newScheduledThreadPool(1)`

Reply:
Yes, this line loads the persistent queue if any, 

Line on 443 starts a timer to periodically save the persistent queue

Reply:
Ok, in my code line 443 launch restore queue first time , inside function there's a check and if persistent isn't enabled, return.

I push all commit always on Friday...

Reply:
> Where is it called? line 443 is `val scheduler = Executors.newScheduledThreadPool(1)`

Sorry, In your code yes, example today in my code there are a couple of new code relate new notifications, but if not work I'll rollback... my lines are different...

Reply:
> Yes, this line loads the persistent queue if any, 
> 
> Line on 443 starts a timer to periodically save the persistent queue
> 
> both should be there
> 
> @fast4x 

Your code work perfectly... it start also if persistent queue is enabled.